runcoms/README.md

```diff
@@ -10,61 +10,199 @@ in */etc* by default.
 
 User-local configuration files have the same name as their global counterparts
 but are prefixed with a dot (hidden). Zsh looks for these files in the path
-stored in the `$ZDOTDIR` environmental variable. However, if said variable is
-not defined, Zsh will use the user's home directory.
+stored in the `$ZDOTDIR` environment variable. However, if said variable is
+not defined, Zsh will use the user's home directory. This is found on the 
+config scripts as `${ZDOTDIR:-$HOME}` and is a `zsh` idiomatic way of doing 
+something like this:
 
-File Descriptions
------------------
-
-The configuration files are read in the following order:
 
-  01. /etc/zshenv
-  02. ~/.zshenv
-  03. /etc/zprofile
-  04. ~/.zprofile
-  05. /etc/zshrc
-  06. ~/.zshrc
-  07. ~/.zpreztorc
-  08. /etc/zlogin
-  09. ~/.zlogin
-  10. ~/.zlogout
-  11. /etc/zlogout
+```bash
+if [[ -n $ZDOTDIR ]]; then
+   DOTFILES_PATH=$ZDOTDIR
+else
+  DOTFILES_PATH=$HOME
+fi
+source $DOTFILES_PATH/.zshenv
 
-### zshenv
+#pro tip! set $ZDOTDIR on /etc/zshenv to ~/.zconf to have all runcoms live there instead of cluttering ~ =)
+```
+the above  is the same as:  `source ${ZDOTDIR:-$HOME}/.zshenv` is equivalent to:
 
-This file is sourced by all instances of Zsh, and thus, it should be kept as
-small as possible and should only define environment variables.
 
-### zprofile
+Sourcing of these confguration files depends mostly on 2 things:
 
-This file is similar to zlogin, but it is sourced before zshrc. It was added
-for [KornShell][1] fans. See the description of zlogin below for what it may
-contain.
+1. Whether you are running a *login shell*
+2. Whether the `RCS` and `GLOBAL_RCS` options are set. They are usually
+   set by default
 
-zprofile and zlogin are not meant to be used concurrently but can be done so.
 
-### zshrc
+File Descriptions
+-----------------
 
-This file is sourced by interactive shells. It should define aliases,
-functions, shell options, and key bindings.
 
-### zpreztorc
+| always runs      | when `GLOBAL_RCS` is set                | when `RCS` is set                 | Purpose                                                                                                                                                                                                  | stuff that usually goes there, and notes                                                                                                                                                                                         |
```

Comment:
This table is a bit wide... I think we could probably drop a column by having `File`, and `Triggers` to replace `always runs`, `when GLOBAL_RCS is set` and `when RCS is set` because only one of those ever has data in it at once.

Reply:
quite right! i thought so too. and good idea i'll make it so the headers go something lije this:

| system/site config (`/etc/zsh/*`)        | User's config (`${ZDOTDIR:-$HOME}`)  | Purpose | Example/Notes    |
|-----------------------------------------|---------------------------------------------------|-------------|-------------------------|
| stuff                                          |     ,+stuff                                                 |       stuff   |  even more stuff                |
| more stuff | user stuff | Purpose stuff | example stuff